src/components/ui/Skeleton.tsx

```
@@ -0,0 +1,13 @@
+import React from 'react';
+import { cn } from '../../utils/cn';
+
+interface SkeletonProps extends React.HTMLAttributes<HTMLDivElement> {}
```

ESLint

[error] An interface declaring no members is equivalent to its supertype.

(@typescript-eslint/no-empty-object-type)